node-repository/src/main/java/com/yahoo/vespa/hosted/provision/node/IP.java

```diff
@@ -325,10 +325,14 @@ public int eventuallyUnusedAddressCount(NodeList nodes) {
             return ipAddresses.protocol;
         }
 
-        public Set<String> getIpSet() {
+        public Set<String> asSet() {
```

Comment:
A Pool is a pool of IP addresses and/or hostnames, which is why this method should be named getIpSet().